File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-1.java
class Solution {
    public int evalRPN(String[] tokens) {
        Deque<Integer> stack = new ArrayDeque<Integer>();
        for(String token : tokens){
            if("+".equals(token) || "-".equals(token) || "*".equals(token) || "/".equals(token)){
                Integer temp = 0;
                Integer el1 = stack.pop();
                Integer el2 = stack.pop();
                Integer result = Integer.MIN_VALUE;
                if("+".equals(token)) {
                    result = el1 + el2;
                }else if("-".equals(token)) {
                    result = el2 - el1;
                }else if("*".equals(token)) {
                    result = el2 * el1;
                }else if("/".equals(token)) {
                    result = el2 / el1;
                }
                stack.push(result);
            }else{
                Integer intVal = Integer.parseInt(token);
                stack.push(intVal);
            }
        };
        return stack.pop();
        
    }
}

File: Data Structures & Algorithms/number-of-one-bits/submission-1.js
class Solution {
    /**
     * @param {number} n - a positive integer
     * @return {number}
     */
    hammingWeight(n) {
        let result = 0;
        while(Number(n) !== 0){
            result = result + (n & 1);
            n >>= 1;
        }
        return result;

    }
}
